modules/PhyVersoBSP/phyverso_config/evConfig.cpp

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#include "evConfig.h"
+#include "phyverso.pb.h"
+#include <fstream>
+#include <iostream>
+
+// for convenience
+using json = nlohmann::json;
+using namespace std;
```

Comment:
At the moment only std::exception doesn't use the std:: namespace, everything else is prefixed